manim/utils/space_ops.py

```diff
@@ -8,6 +8,9 @@
     "quaternion_conjugate",
     "rotate_vector",
     "thick_diagonal",
+    "rotation_matrix_transpose_from_quaternion",
+    "rotation_matrix_from_quaternion",
+    "rotation_matrix_transpose",
```

Comment:
How useful are these function for the Users?

Reply:
Has been resolved (removed)